android/app/src/main/java/org/fossasia/openevent/common/date/DateConverter.java

```diff
@@ -169,4 +169,11 @@ public static String getRelativeTimeFromUTCTimeStamp(String timeStamp) throws Da
                 (timeCreatedDate.toInstant().toEpochMilli()),
                 System.currentTimeMillis(), android.text.format.DateUtils.SECOND_IN_MILLIS);
     }
+
+    public static String getRelativeTimeFromOffsetDateTime(String timeStamp) throws DateTimeParseException {
+        ZonedDateTime timeCreatedDate = ZonedDateTime.from(DateTimeFormatter.ISO_OFFSET_DATE_TIME.parse(timeStamp));
+        return (String) android.text.format.DateUtils.getRelativeTimeSpanString(
+                (timeCreatedDate.toInstant().toEpochMilli()),
+                System.currentTimeMillis(), android.text.format.DateUtils.SECOND_IN_MILLIS);
+    }
```

Comment:
We cannot unit test this. Any import of android.* should be forbidden for use in core classes. Create another class AndroidDateConverter, and add this method there, use any method present in DateConverter as a helper

Reply:
The ```android.text.format.DateUtils.SECOND_IN_MILLIS``` is just ```1000L```. Can't i just add a constant in this DateConverter with value of ```1000L``` ?